tests/expectations/tests/repr-align.rs

```diff
@@ -0,0 +1,35 @@
+/* automatically generated by rust-bindgen */
+
+#![allow(dead_code, non_snake_case, non_camel_case_types, non_upper_case_globals)]
+
+#[repr(C)]
+#[repr(align(8))]
+#[derive(Debug, Default, Copy, Clone)]
+pub struct a {
+    pub b: ::std::os::raw::c_int,
+    pub c: ::std::os::raw::c_int,
+    pub __bindgen_align: [u64; 0usize],
```

Comment:
Do we still need to generate this, then?

Reply:
Not really, but it doesn't hurt and avoids having the extra codepath and conditionals.